### ISSUE TITLE
Restaurant Unit Tests

### ISSUE THREAD

**Issue by shanemileham** [usc-csci201-fall2013/simcity201#56]
In class today, we talked about unit tests for the individual restaurants which have their own version of each class (multiple waiters, hosts, etc.), and @crowley1958 ventured that we only would have to do unit tests for the shared data waiter cases because that is the only new additions to our restaurants. This makes sense because we've been checked off through V2.2 and only really need to test our new functionality to make sure it works.

@dwilczyn - This is correct, I assume?

**Comment by dwilczyn** [comment]
Yes, I only want NEW code tested. For the restaurant that mostly means:
1) Producer Consumer
2) Interactions with the NEW and improved markets
